src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -2134,4 +2136,116 @@ public EventsFilterParam(String name, String value) {
     }
   }
 
+  Volume createVolume() throws DockerException, InterruptedException;
+
+  Volume createVolume(Volume volume) throws DockerException, InterruptedException;
+
+  Volume inspectVolume(String volumeName) throws DockerException, InterruptedException;
+
+  void removeVolume(Volume volume) throws DockerException, InterruptedException;
```

Comment:
I lean towards keeping the number of methods smaller and just keeping the`public void removeVolume(final String volumeName)`.